### PR TITLE
issue #12: refactor code

### PR DESCRIPTION
**Pull request by Subham5401**
### Summary 🎯

I've made some changes to the codebase to improve its readability and maintainability.

### Changes 🔁

- Removed unnecessary comments from the codebase

I believe these changes will make it easier for others to understand and work with this codebase in the future. Let me know if there are any questions or concerns.

### CODE REVIEW

File: src/widgets/index.tsx [hunk 1]
```diff
@@ -283,120 +257,107 @@ async function setAsQueue(plugin: ReactRNPlugin) {
...
-
-async function onDeactivate(_: ReactRNPlugin) {
-  console.log('deactivating!');
-}
 
-declareIndexPlugin(onActivate, onDeactivate);
+  declareIndexPlugin(onActivate, onDeactivate);
 
-export { sendHeartbeat };
+  export { sendHeartbeat };
```
Comment (coldenate) [inline src/widgets/index.tsx:363]: Is there a formatting confliction? Why is there an added space here?

**Review by coldenate — COMMENTED** [review]
Hey @Subham5401, 👋  thanks for your hard work. 

I'm noticing a couple of things that I think *I* overlooked and are my error. This is because the code is not ready to be refactored due to many comments being todo statements and infant functions. 

- The code is not ready to be refactored. I will close this issue for now, and let you know when it is reopened.
   - I realized it was probably confusing to see my personal function notes in there. thats my bad 😭 
- Every single comment was removed, which I believe is a bit more destructive. I think if you revisit this it would be worth attempting to modify or improve the placement.
- Let's leave the `TODO:`s for now. 

I'm also noticing your changes break the file? I'm not sure if it's a conflicting encoding or tab usage, but the functions seem to incorrect scopes and the braces are mismatched.

File: src/widgets/index.tsx (hunk 1 above)
Comment (Subham5401) [reply src/widgets/index.tsx:363]: Hi @coldenate, it's probably because of the code formatting tool I use. I did not realize.

**Comment by Subham5401** [comment]
Hi @coldenate, I would appreciate it if you could explain the purpose of your comments so that I can better understand which ones to keep. I'm worried that the changes I made may not meet your expectations. I'm going to close this PR for now, but I hope to have another chance to work on this issue in the future. Thank you.

**Comment by coldenate** [comment]
@Subham5401 You're good! I appreciated the effort 💯!! I'm pretty new to public issues, so I was not fully aware of the effects of help wanted. If the issue is not resolved in a future commit, I'll re-mention the issue when the code is at a point that I am comfortable being refactored. I left a lot of comments I needed but that shouldn't be in the Final Cut. Thanks for considering!

**Pull request closed without merging**